lib/instrumentation/modules/express.js

```diff
@@ -47,7 +47,9 @@ module.exports = function (express, agent, version, enabled) {
             if (!layer.route && layerPath && typeof next === 'function') {
               safePush(req, symbols.expressMountStack, layerPath)
               arguments[2] = function () {
-                req[symbols.expressMountStack].pop()
+                if (!(req.route && arguments.length && arguments[0] instanceof Error)) {
```

Comment:
What's the reason for the `arguments.length` check?

Reply:
Oh, the `arguments.length`  code is not required. I will remove it.